climada_petals/hazard/test/test_tc_rainfield.py

```diff
@@ -65,28 +75,48 @@ def test_set_one_pass(self):
         self.assertEqual(tc_haz.event_id[0], 1)
         self.assertEqual(tc_haz.event_name, ['1951239N12334'])
         self.assertTrue(np.array_equal(tc_haz.frequency, np.array([1])))
-        self.assertTrue(isinstance(tc_haz.intensity, sparse.csr.csr_matrix))
-        self.assertTrue(isinstance(tc_haz.fraction, sparse.csr.csr_matrix))
-        self.assertEqual(tc_haz.intensity.shape, (1, 296))
+        self.assertTrue(isinstance(tc_haz.fraction, sparse.csr_matrix))
         self.assertEqual(tc_haz.fraction.shape, (1, 296))
+        self.assertIsNone(tc_haz._get_fraction())
 
-        self.assertAlmostEqual(tc_haz.intensity[0, 100], 99.7160586771286, 6)
-        self.assertAlmostEqual(tc_haz.intensity[0, 260], 33.2087621869295)
-        self.assertEqual(tc_haz.fraction[0, 100], 1)
-        self.assertEqual(tc_haz.fraction[0, 260], 1)
+        self.assertTrue(isinstance(tc_haz.intensity, sparse.csr_matrix))
+        self.assertEqual(tc_haz.intensity.shape, (1, 296))
+        self.assertEqual(tc_haz.intensity.nonzero()[0].size, 296)
+        self.assertAlmostEqual(tc_haz.intensity[0, 100], 98.61962462510677, 6)
+        self.assertAlmostEqual(tc_haz.intensity[0, 260], 27.702589231065055)
```

Comment:
Changing the R-CLIPER model was not the purpose of this PR. Still, as mentioned in the comments above we found and fixed a bug in line 244 of the implementation along the way. That's why we also had to change the intensity values in the unit tests for the R-CLIPER model.